.gitignore

```diff
@@ -24,3 +24,7 @@ vendor/*
 # Blender build artefacts:
 blender/extension.zip
 blender/extension/wheels/*
+
+# Artifact workflow
+dag.svg
+filegraph.svg
```

Comment:
You don't need to gitignore these files from the workflow, since they don't go in git. If you want to generate them for yourself locally and ignore them, that's maybe okay, but if you don't ignore them here, that's not going to make problems for the github action.

Reply:
Yeah, if you want to make it easier for yourself by ignoring these, that's generally not a big deal, your call.

Reply:
Okay, considering you accidentally added the filegraph to the commit, maybe it's better to ignore them after all 😅. Instead of ignoring them in the root, however, I put them in `04_extra`.